zkevm-circuits/src/evm_circuit/execution/stop.rs

```diff
@@ -54,7 +57,11 @@ impl<F: Field> ExecutionGadget<F> for StopGadget<F> {
         );
 
         // Call ends with STOP must be successful
-        cb.call_context_lookup(false.expr(), None, CallContextFieldTag::IsSuccess, 1.expr());
+        cb.call_context_lookup_read(
+            None,
+            CallContextFieldTag::IsSuccess,
+            Word::from_lo_unchecked(1.expr()),
```

Comment:
Idea: implement `Word::one()` and `Word::zero()`. I found 5 instances of this pattern.  Maybe it's not enough instances to justify a new method :thinking:

Reply:
These 2 method already there 😅 ~~ I just replace all of them so it's consistent now